### PR TITLE
fix(ImageConfig): Caption alignment

### PR DESCRIPTION
**Pull request by thiskevinwang**
## 🔗 Relevant links

- [Preview link](https://dev-portal-git-BRANCH_NAME-hashicorp.vercel.app/PATH_TO_VIEW) 🔎
- [Asana task](url) 🎟️

## 🧪 Testing

https://dev-portal-git-kevin-image-config-caption-alignment-hashicorp.vercel.app/swingset/components/imageconfig

### CODE REVIEW

File: src/components/image/image.module.css [hunk 1]
```diff
@@ -1,38 +1,37 @@
 .root {
-  background-color: white;
-  box-shadow: var(--token-surface-base-box-shadow);
-  border-radius: 6px;
-  display: flex;
-  flex-wrap: wrap;
-  overflow: hidden;
-  text-align: center;
```
Comment (thiskevinwang) [inline src/components/image/image.module.css:8]: removed `text-align`

File: src/components/image/image.module.css [hunk 2]
```diff
@@ -1,38 +1,37 @@
...
-  justify-content: center;
-  margin: 24px auto;
+	background-color: white;
+	box-shadow: var(--token-surface-base-box-shadow);
+	border-radius: 6px;
+	display: flex;
+	flex-wrap: wrap;
+	overflow: hidden;
+	justify-content: var(--justify, center);
```
Comment (thiskevinwang) [inline src/components/image/image.module.css:8]: added variable usage to `justify-content`

File: src/components/image/image.module.css [hunk 3]
```diff
@@ -1,38 +1,37 @@
...
+	margin: 24px auto;
 
-  &.noMargin {
-    margin: 0;
-  }
+	&.noMargin {
```
Comment (thiskevinwang) [inline src/components/image/image.module.css:11]: Rest is auto-formatting (space-to-tabs)

**Review by kendallstrautman — APPROVED** [review]
LGTM! 👏

**Pull request closed without merging**